static/src/javascripts/es6/projects/common/modules/crosswords/anagram-helper/shuffler.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import _ from 'common/utils/_';
+
+const round = x => Math.round(x * 100) / 100;
+
+export default class Shuffler extends React.Component {
+    getLetters () {
+        const entries = _.chain(this.props.entries)
+            .map(entry => entry.value.toLowerCase())
+            .filter(entry => _.contains(this.props.word.map(l => l.toLowerCase()), entry))
```

Comment:
good spot. just realised I made the word lowercase further up the chain, so the map can be removed altogether